nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="bg-black-core text-white text-center h-[550px] lg:h-[750px]">
+    <div
+      class="container flex gap-6 lg:gap-[4.5rem] flex-col pt-12 xs:pt-14 sm:pt-[4.5rem] lg:!pt-[7.5rem] xl:!pt-[6rem] lg:px-[12rem]"
+    >
+      <div class="header-2">Our Success Stories</div>
+      <div class="flex flex-col gap-6 lg:gap-10">
+        <img
+          :src="quoteimage"
+          class="mx-auto w-[2.5rem] lg:w-[3.5rem] h-full object-contain"
+          alt="quote-image"
+        />
+        <div class="swiper-content">
+          <swiper
+            :slidesPerView="1"
+            :autoplay="{
+              delay: 3000,
+              disableOnInteraction: false,
+            }"
+            :speed="3000"
+            :loop="true"
+            :spaceBetween="20"
+            :pagination="pagination"
+            :modules="modules"
+            class="swiper-container"
+          >
+            <swiper-slide v-for="(client, index) in clients" :key="index">
+              <div class="flex flex-col gap-6 lg:gap-10">
+                <div class="sub-h1-semibold">"{{ client.review }}"</div>
```

Comment:
all reviews starting line should be same its not like one start from above and one from below

Reply:
Unable to the point can you please elaborate more

Reply:
means top margin should be same for all reviews refer mobile app development what our client says section